src/streamlink/plugins/afreeca.py

```diff
@@ -120,14 +125,14 @@ def _get_channel_info(self, broadcast, username):
         res = self.session.http.post(self.CHANNEL_API_URL, data=data)
         return self.session.http.json(res, schema=self._schema_channel)
 
-    def _get_hls_key(self, broadcast, username, quality):
+    def _get_hls_key(self, broadcast, username, quality, stream_password):
         data = {
             "bid": username,
             "bno": broadcast,
             "from_api": "0",
             "mode": "landing",
             "player_type": "html5",
-            "pwd": "",
+            "pwd": stream_password,
```

Comment:
`stream_password` will be `None` if unset. Is that still fine?

Reply:
If it's "" when None, that's fine. If it's not "" when None, I think we should handle it differently.

Reply:
```suggestion
            "pwd": stream_password or "",
```